policyengine_us/variables/gov/states/ct/tax/income/ct_non_refundable_credits.py

```diff
@@ -8,3 +8,5 @@ class ct_non_refundable_credits(Variable):
     unit = USD
     definition_period = YEAR
     defined_for = StateCode.CT
+
+    adds = ["ct_property_tax_credit"]
```

Comment:
make a yaml file called non_refundable in the credit folder and add the credit to a list